### PR TITLE
Feature/derp 450 civic uk proof of concept

### PR DESCRIPTION
**Pull request by lukewhitmore**
Issue [here](https://london.atlassian.net/browse/DERP-450)

I wasn't sure whether this should be merged, as I heard there might be a further implementation ticket down the line.

Feels to me that there's no harm in adding this basics setup now though.

We obviously won't have the API secrets injected until Scott is back - but once they are we should be able to test on sub-sites via GLA architecture.

### CODE REVIEW

File: config/install/google_tag.settings.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    nonGoogleScripts
+    nonGoogleIframes
+  blacklist_classes: |-
+    customScripts
+    customPixels
+  include_environment: false
+  environment_id: ''
+  environment_token: ''
+_core:
+  default_config_hash: 65d5DfK5p2hQDpBskbvCIj2hSEqw_Odm2e_d46BH1Lk
```
Comment (philwolstenholme) [inline config/install/google_tag.settings.yml:38]: We should remove the `_core.default_config_hash` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/civiccookiecontrol.cookiecategory.site_metrics.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+uuid: 46ce5c8d-91f4-4d23-9962-2e9d1ebdae48
```
Comment (philwolstenholme) [inline config/install/civiccookiecontrol.cookiecategory.site_metrics.yml]: We should remove the `uuid` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/civic_govuk_cookiecontrol.settings.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+govuk_cookiecontrol_change_cookie_settings_link_text: 'change your cookie settings'
+govuk_cookiecontrol_change_cookie_settings_suffix: 'at any time'
+govuk_cookiecontrol_hide_button_text: Hide
+govuk_cookiecontrol_optional_cookie_text: 'Optional cookie settings'
+govuk_cookiecontrol_saved_settings_text: 'Your settings have been saved.'
+govuk_cookiecontrol_save_and_continue: 'Save and continue'
+govuk_cookiecontrol_allow_cookies_question_prefix: 'Can we use'
+govuk_cookiecontrol_allow_cookies_question_suffix: 'cookies to help us improve the service?'
+_core:
+  default_config_hash: _IOp7sGqTREbYGmk1Kl1yUMvTr5_4lPNESKb2pfe2iE
```
Comment (philwolstenholme) [inline config/install/civic_govuk_cookiecontrol.settings.yml:13]: We should remove the `_core.default_config_hash` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/civiccookiecontrol.iab.yml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+iabIabLinkText: 'https://advertisingconsent.eu/'
+iabPanelBackText: 'Back to All Categories'
+iabVendorTitleText: 'Ad Vendors'
+iabVendorConfigureText: 'Show Ad Vendors'
+iabVendorBackText: 'Back to Ad Vendor Purposes'
+iabAcceptAllText: 'Accept All'
+iabRejectAllText: 'Reject All'
+iabBackText: Back
+_core:
+  default_config_hash: QyUND8ghCDyaK1Wh9poHBGbyc74NCqNnaS7-oKLYX6c
```
Comment (philwolstenholme) [inline config/install/civiccookiecontrol.iab.yml]: We should remove the `_core.default_config_hash` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/civiccookiecontrol.iab2.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+iabUsesNonCookieAccessTrue: 'Uses other means for storing information, eg. localstorage'
+iabUsesNonCookieAccessFalse: 'Only uses cookies to store information'
+iabStorageDisclosures: 'Device Storage Duration & Access Disclosure'
+iabDisclosureDetailsColumn: 'Storage Details'
+iabDisclosurePurposesColumn: Purposes
+iabSeconds: seconds
+iabMinutes: minutes
+iabDays: days
+iabHours: hours
+_core:
```
Comment (philwolstenholme) [inline config/install/civiccookiecontrol.iab2.yml]: We should remove the `_core.default_config_hash` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/civiccookiecontrol.settings.yml [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+civiccookiecontrol_ccpa_stmt_date: ''
+civiccookiecontrol_ccpa_stmt_url: ''
+civiccookiecontrol_show_vendors: 'Show vendors within this category'
+civiccookiecontrol_third_party_cookies: 'This vendor may set third party cookies.'
+civiccookiecontrol_read_more: 'Read more'
+civiccookiecontrol_notify_font_color: ''
+civiccookiecontrol_notify_bg_color: ''
+civiccookiecontrol_close_text: ''
+civiccookiecontrol_close_bg_color: ''
+_core:
```
Comment (philwolstenholme) [inline config/install/civiccookiecontrol.settings.yml]: We should remove the `_core.default_config_hash` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

File: config/install/google_tag.container.gla_estate.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+uuid: 59d8df90-6acc-4cd7-8e38-8c7300e4df8f
```
Comment (philwolstenholme) [inline config/install/google_tag.container.gla_estate.yml:1]: We should remove the `uuid` here. Here's a script: https://ctidigital.slack.com/archives/C01GPQ0JXST/p1624974769038700

**Review by philwolstenholme — CHANGES_REQUESTED** [review]

File: config/install/civiccookiecontrol.settings.yml (hunk 6 above)
Comment (lukewhitmore) [reply config/install/civiccookiecontrol.settings.yml]: This is already removed!

File: config/install/civiccookiecontrol.iab2.yml (hunk 5 above)
Comment (lukewhitmore) [reply config/install/civiccookiecontrol.iab2.yml]: This is already removed!

File: config/install/civiccookiecontrol.cookiecategory.site_metrics.yml (hunk 2 above)
Comment (lukewhitmore) [reply config/install/civiccookiecontrol.cookiecategory.site_metrics.yml]: This is already removed!

File: config/install/civiccookiecontrol.iab.yml (hunk 4 above)
Comment (lukewhitmore) [reply config/install/civiccookiecontrol.iab.yml]: This is already removed!

File: config/install/civic_govuk_cookiecontrol.settings.yml (hunk 3 above)
Comment (lukewhitmore) [reply config/install/civic_govuk_cookiecontrol.settings.yml]: Done!

File: config/install/google_tag.settings.yml (hunk 1 above)
Comment (lukewhitmore) [reply config/install/google_tag.settings.yml]: Done!

**Comment by lukewhitmore** [comment]
@philwolstenholme Hey .. those should all be removed now - I think one of my commits hadn't refreshed, but then there _were_ a few others that I'd missed also!

**Review by philwolstenholme — APPROVED** [review]

**Pull request merged**